components/automate-dex/web/templates/password.html

```diff
@@ -39,15 +34,15 @@
         <div class="theme-form-label">
           <label for="userid">Username</label>
         </div>
-        <input tabindex="1" required id="login" name="login" type="text" class="theme-form-input" {{ if .Username }} value="{{ .Username }}" {{ else }} autofocus {{ end }}/>
+        <input autocomplete="off" required id="login" name="login" type="text" class="theme-form-input" {{ if .Username }} value="{{ .Username }}" {{ else }} autofocus {{ end }}/>
       </div>
       <div class="theme-form-row">
         <div class="theme-form-label">
           <label for="password">Password</label>
         </div>
-        <input tabindex="2" required id="password" name="password" type="password" class="theme-form-input" {{ if .Invalid }} autofocus {{ end }}/>
+        <input autocomplete="off" required id="password" name="password" type="password" class="theme-form-input" {{ if .Invalid }} autofocus {{ end }}/>
       </div>
-      <button tabindex="3" type="submit" class="dex-btn theme-btn--primary">Sign In</button>
+      <button autocomplete="off" type="submit" class="dex-btn theme-btn--primary">Sign In</button>
```

Comment:
Does `autocomplete="off"` actually do something in a button...? 🤔

Reply:
Sorry sorry that was a mistake, y'all can take that out.